R-proj/src/Makevars

```diff
@@ -1,4 +1,4 @@
-PKG_CPPFLAGS=-I../../external/boost -I../../external/LPsolve_src/run_headers -I../../external/minimum_ellipsoid -I../../include 
+PKG_CPPFLAGS=-I../../external/boost -I../../external/LPsolve_src/run_headers -I../../external/minimum_ellipsoid -I../../include -I../../include/convex_bodies/spectrahedra
```

Comment:
please do not include any subdirectory of `/root/include in makevars.

Reply:
this will change when i merge some changes from previous pull request